README.md

```diff
@@ -1,4 +1,5 @@
 # healthchecks
+[![Build Status](https://travis-ci.org/andela/healthchecks-f-society.svg?branch=develop)](https://travis-ci.org/andela/healthchecks-f-society)
```

Comment:
Remove the query parameter specifying the `develop` branch. So as to have the badge update based off the last build.

i.e 
```markdown
[![Build Status](https://travis-ci.org/andela/healthchecks-f-society.svg)](https://travis-ci.org/andela/healthchecks-f-society)
```